# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/target/server/etc_profile.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- rock-generic_etc_profile	2003-05-18 00:12:08.000000000 +0200
+++ rock-desktop_etc_profile	2003-05-18 00:54:13.000000000 +0200
@@ -43,7 +43,8 @@
 #
 if [ "$PS1" ] ; then
 	PS1='$ ' ; PS2='> ' ; PS4='+ '
-	[ "$BASH_VERSION" ] && PS1='\u@\h:\w\$ '
+	color="1;32m" ; [ "$UID" = 0 ] && color="0;31m"
+	[ "$BASH_VERSION" ] && PS1="\[\033[$color\]\u@\h:\w\\$\[\033[0m\] "
 	[ "$KSH_VERSION" -a   -w / ] && PS1='$USER:$PWD# '
 	[ "$KSH_VERSION" -a ! -w / ] && PS1='$USER:$PWD# '
 fi
